Tests/Test_MotorController.c

```diff
@@ -1,28 +1,160 @@
 /**
- * File for testing Motor Controller and verifying CAN2 functionality
+ * File for testing Motor Controller and verifying CAN3 functionality
  * 
- * The user sets "id" to be the desired Motor Controller command ID 
- * and "data[8]" to be the desired message. The test outputs the id 
- * and message that is received by the Motor Controller. "Success" 
- * is 1 if the message recieved is the expected message.
+ * 
+ * This file spins the motor up to .5amps for 200ms, then 1 amp for 200ms, 
+ * and then coasting for 200ms and repeats. 
  */ 
 #include "common.h"
 #include "config.h"
 #include "MotorController.h"
+#include "os.h"
+#include "Tasks.h"
+
+static OS_TCB Task1TCB;
+static CPU_STK Task1Stk[128];
+static car_state_t car_state;
+
+
+#define BUSCURRENT 10
+
+void Task1(void* arg){
+    CPU_Init();
+    OS_CPU_SysTickInit(SystemCoreClock / (CPU_INT32U) OSCfg_TickRate_Hz);
+
+    OS_ERR err;
+
+
+    MotorController_Init(1.0f);
+    float unatttainable_Velocity;
+    float desiredCurrent; 
+    float currentSetPoint;
+
+
 
-int main(){
+    while(1){
+        unatttainable_Velocity = 2500.0f;
+
+        desiredCurrent = 1.0f; 
+        currentSetPoint = desiredCurrent/BUSCURRENT;
+        for (int i=0; i<30; i++) { // Run for ~6 seconds
+            MotorController_Drive(unatttainable_Velocity,currentSetPoint);
+            OSTimeDlyHMSM(0,0,0,200,OS_OPT_TIME_HMSM_STRICT, &err);
+            if(err != OS_ERR_NONE){
+                volatile int x = 0;
+                while(1){
+                    x++;
+                }
+            }
+        }
+
+        desiredCurrent = 5.0f; //1 amps worth
```

Comment:
Is this comment accurate? If so, having `5.0f` correspond to 1 amp is a bad interface